.github/workflows/regression.yml

```diff
@@ -85,12 +87,36 @@ jobs:
           cp -r ${{format('Base{0}', matrix.tag)}}/. install/bin/
           cp -r ${{format('Reverse{0}', matrix.tag)}}/. install/bin/
           cp -r ${{format('Forward{0}', matrix.tag)}}/. install/bin/
-          chmod a+x install/bin/*               
+          chmod a+x install/bin/*
+      - name: Run Tests in Container
+        uses: docker://su2code/test-su2:20200303
+        with:
+          args: -b ${{github.ref}} -t develop -c develop -s ${{matrix.testscript}}
+  regression_tests_omp:
```

Comment:
I am gonna simplify the script a little bit. I don't think its necessary to copy all that stuff. You are only doing it because the Reverse/Forward binaries are not available for the BaseOMP case, right? 

I am also surprised that OpenMP is installed in the docker container ...

Reply:
Yep.
I don't think one installs OpenMP, it comes with the compiler.